Playwright

ealonso opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

sandrodw3 commented:
Pull request submitted to https://github.com/ealonso/liferay-portal/pull/2462. [See changes here.](https://github.com/sandrodw3/liferay-portal/compare/c4f07efd2062~1...c4f07efd2062) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

vendeltoreki committed:
LPD-28243 Use batch delete to remove DispatchLog records

vendeltoreki committed:
LPD-28243 Add test case for deleting DispatchLog records in batch and not one-by-one

gabrielwas committed:
LPD-28585 Expose createNotificationRecipientSetting

gabrielwas committed:
LPD-28585 Move and adapt notification object action tests to ObjectActionLocalServiceTest

gabrielwas committed:
LPD-28585 Add notification object action test for commerce order

gabrielwas committed:
LPD-28585 Add old value test for commerce order notification action

gabrielwas committed:
LPD-28585 Fix commerce order predefined values adding the new required fields

gabrielwas committed:
LPD-28585 Already covered by integration tests

vendeltoreki committed:
LPD-28243 Simplify and use the right method clear the cache

vendeltoreki committed:
LPD-28243 Move the delete method to DispatchLogLocalServiceImpl

izaera committed:
LPD-30580 Use esbuild for frontend-js-svg4everybody-web's build

victorg1991 committed:
LPD-29876 Don't depend on the whole store

victorg1991 committed:
LPD-29876 Load the panels directly since with ESM all the js is executed at first so there is no need to postpone it

victorg1991 committed:
LPD-29876 Adapt Toolbar Section to render it directly

victorg1991 committed:
LPD-29876 Not needed

victorg1991 committed:
LPD-29876 Update test

victorg1991 committed:
LPD-29876 Adapt

gislayne-vitorino committed:
LPD-30056 Add playwright test for PublicationsConflict#ResolveDeletionModificationConflictPublicationsByDiscarding

Commit:
LPD-30131 Add Marketplace Helpers

ling-alan-huang committed:
LPD-30270 Skip when the variable is in triple quotes

Commit:
LPD-30131 Refactor to use helpers

Commit:
LPD-30131 Add Customer Dashboard App Details

Commit:
LPD-30131 Improving code

ling-alan-huang committed:
LPD-30270 Apply to *.macro

ling-alan-huang committed:
LPD-30270 Skip when the line contains curl

ling-alan-huang committed:
LPD-30270 Skip if condition

ling-alan-huang committed:
LPD-30270 SF, inline

juanjofgliferay committed:
LPD-30224 - Refactor. Migrate Data Set tests to Playwright

Remove Data Set Poshi tests related to the Data Set table sorting

juanjofgliferay committed:
LPD-30224 - SF

ling-alan-huang committed:
LPD-30270 Skip when it's a global variable

matyaswollner committed:
LPD-30006 - Refactor LocalFile.InstanceSettings#EditDefaultUserAssociationsUserGroups into a PLaywright test

matyaswollner committed:
LPD-30006 - Removing poshi test InstanceSettings#EditDefaultUserAssociationsUserGroups

ling-alan-huang committed:
LPD-30270 Skip when it is in if block

ling-alan-huang committed:
LPD-30270 Auto SF

gislayne-vitorino committed:
LPD-30056 Allow more than one Publication to be created by modifying externalReferenceCode to have a random value

gabrielwas committed:
LPD-28585 baseline

gislayne-vitorino committed:
LPD-30056 Remove previous ResolveDeletionModificationConflictPublicationsByDiscarding test

gabrielwas committed:
LPD-28585 Create new test for notification object actions within system objects

gabrielwas committed:
LPD-28585 Checks on the notification not needed. In this suite (ObjectActionLocalServiceTest) only the trigger actions will be checked, and not the consistency of the notification message

gabrielwas committed:
LPD-28585 Changes in Principal not needed

Commit:
LPD-30434 Add Solution Details page test

Commit:
LPD-30434 Add getTaxonomyCategoryByVocabularyId and getTaxonomyVocabularyBySiteId

Commit:
LPD-30434 Improving code

Commit:
LPD-30131 Add contactSupportUrl property

Commit:
LPD-30562 Improve helper

brittneyq committed:
LRCI-4312 Move fragment test to correct directory

ealonso committed:
LPD-X Moves tests to publications team

marcoscv-work committed:
LPD-30511 - fix user styles

victorg1991 committed:
LPD-29876 Include title

victorg1991 committed:
LPD-29876 AutoSF

boton committed:
Revert "LPD-28331 Set testIdAttribute: 'data-qa-id' to all tests at once"

This reverts commit 62fdc150aec50a70247fba9e00a0dbfe3f444960.

juanjofgliferay committed:
LPD-30224 - Refactor. Ensure Data Set creation order

boton committed:
LPD-30245 Avoid use of getByTestId in general utils

boton committed:
LPD-30245 Add extra waitFor expecting the reset button is ready

boton committed:
LPD-30245 Add missing id to link with the label and use getByLabel in test instead of getByTestId

ling-alan-huang committed:
LPD-30270 SF, do not inline

adolfopa committed:
LPD-30519 Articles may have empty/null content

It is possible to create a journal article with empty content. The
main use case is when defining default values for a structure; that is
implemented by creating a hidden journal article containing those
default values. In this case, it is perfectly sensible for the content
field to be empty/null.

adolfopa committed:
LPD-30519 Add integration test

adolfopa committed:
LPD-30519 Rename test class to match class under test

juanjofgliferay committed:
LPD-30523 - Refactor. Remove Poshi test covered by Playwright

Case covered by /frontend-data-set-admin-web/tests/data-set-admin/selectionFilters.spec.ts#Can create a selection filter from picklist source

boton committed:
LPD-30245 Avoid use of getByTestId

boton committed:
LPD-30245 Use clickAndExpectToBeHidden to avoid flakiness

Commit:
LPD-30131 Create Marketplace Helper fixture

juanjofgliferay committed:
LPD-30224 - Skip default sort check

ling-alan-huang committed:
LPD-30270 Check the following code in case we use the same variable two or more times

brbalazs committed:
LPD-30009 - Refactor poshi tests

brbalazs committed:
LPD-30009 - Delete poshi tests

brbalazs committed:
LPD-30009 - Correction of permission handling business logic

marcosapmf committed:
LPD-30450 check if there is other property than Beryl Commerce to delete (fix tearDown)

marcosapmf committed:
LPD-30450 test fix

Commit:
LPD-30131 Add types and remove false constant

Commit:
LPD-30131 Move folders

MarinhoFeliphe committed:
LPD-30566 Delete all NotificationRecipientSetting with the name as usePreferredLocaleForGuestUsers

marcosapmf committed:
LPD-30450 use ClayButton when there is no property created

juanjofgliferay committed:
LPD-30224 - Refactor. Ensure FDS use a default sort even if it is not marked as active

juanjofgliferay committed:
LPD-30224 - Enable default test case

juanjofgliferay committed:
LPD-30224 - SF

smottal committed:
LPD-30578 Fix test timeout

brianchandotcom committed:
LPD-30270 prep next

adolfopa committed:
LPD-30456 Add additional assertion to existing tests

brianchandotcom committed:
LPD-30270 prep next

izaera committed:
LPD-30580 Add support for building projects with a custom esbuild

izaera committed:
LPD-30580 Support build:custom target in source formatter

ealonso committed:
LPD-28940 Extract code into a new method just to reuse it

juanjofgliferay committed:
LPD-30224 - Refactor. Revert to simple await Data Set creation.

ealonso committed:
LPD-28940 Pass layout and calculate layoutPageTemplateStructure, in that way we can reuse the same logic for testing draft layouts

adolfopa committed:
LPD-30519 Same for other test classes in package

CrescenzoRegaLF committed:
LPD-20696 Add ObjectValidationRuleEngine

izaera committed:
LPD-28762 SF

4lejandrito committed:
LPD-28872 Fixing NPE caught by tests

https://testray.liferay.com/#/project/34754/routines/34756/build/31352205/case-result/31364382

ling-alan-huang committed:
LPD-29729 Auto SF

ling-alan-huang committed:
LPD-1909 Auto SF

ling-alan-huang committed:
LPD-29968 Auto SF

ealonso committed:
LPD-28940 Adds new test to cover LPP-52327 where if we have a draft layout and the serviceContext.getUserId is not set the upgrade process fail with a null pointer exception

marianoalvarosaiz committed:
LPD-29351 Take into account the changes AM->PM and PM-AM.

boton committed:
LPD-30245 Set missing testIdAttribute: 'data-qa-id' in search-experiences-web and journal-web projects

ambrinchaudhary committed:
LPD-30595 Disable Iframe Sanitizer to add videos. This was done for security reasons under LPD-15081, and users must set the sandbox attributes correctly

ambrinchaudhary committed:
LPD-30595 Remove quarantine test

adolfopa committed:
LPD-30615 Ignore expected warn logs during upgrade

We cannot use the `@ExpectedLogs` annotation because the upgrade class
is not visible from the test module (and so we cannot reference it in
the `loggerClass` annotation attribute).

brbalazs committed:
LPD-30009 - portal-ext.properties added

brianchandotcom committed:
LPD-28243 SF

brianchandotcom committed:
LPD-20696 New pattern

brianchandotcom committed:
LPD-20696 SF

brianchandotcom committed:
LPD-20696 SF

brianchandotcom committed:
LPD-20696 Make a BaseObjectValidationRuleEngineImpl

joseabelenda committed:
LRSD-5048 Order by c_buildId_ to improve performance

joseabelenda committed:
LRSD-5048 My bad

rafaelfs98 committed:
LRSD-5046 History results list is not showed properly

rafaelfs98 committed:
LRSD-5030 Number of tests doesn’t match

marcosapmf committed:
LPD-30450 update snapshots

jonathanmccann committed:
LPD-30691 Add additional locator in order to find workflow status label

JerryAdrianoo committed:
Revert "LPD-29780 SF"

This reverts commit e0983c1e44e6be44a811ef65210d9ef64b471d14.

JerryAdrianoo committed:
Revert "LPD-29780 SF"

This reverts commit 4129cd18b6bbfd0c2c4cdaa8a14f4de67a2b8790.

JerryAdrianoo committed:
Revert "LPD-29780 Rename"

This reverts commit 4505e51282a240167ffc4b0ba63583df0cd4b4ec.

JerryAdrianoo committed:
Revert "LPD-29780 Not needed"

This reverts commit 882d608cad764ff7714b6b21b90f16b0368544d2.

JerryAdrianoo committed:
Revert "LPD-29780 By default it is 1 time"

This reverts commit 43792c4053f22c760061e98876306fd6b1d3fcd6.

JerryAdrianoo committed:
Revert "LPD-29780 Call the protected methods directly, as we can do it for being a unit test of the same package"

This reverts commit cef2d5f1e96f528ae989401a3b862f16c67fb699.

JerryAdrianoo committed:
Revert "LPD-29780 Delegate initialization to Mockito"

This reverts commit 9c55f14867902778a8b02d83d68d00daf2ca0167.

JerryAdrianoo committed:
Revert "LPD-29780 Add unit test to verify whether all types of jobs are scheduled on a master node of a cluster"

This reverts commit 81178178c0a0390a142708324d006bc5e3352be4.

JerryAdrianoo committed:
Revert "LPD-29780 Schedule Single Node (Memory Clustered and Persisted) jobs on master node when the portal starts up"

This reverts commit b1b2854b83fd67c9ff9182708119fdbf4e37ba07.

Commit:
LPD-29589 Add a step to save the dynamic segment before trying to navigate to the segment list

joseabelenda committed:
LRSD-5048 New query for totalCount to improve the performance

jonathanmccann committed:
LPD-30815 Use the correct parameter names from 'message_boards/select_category.jsp'

jonathanmccann committed:
LPD-30815 Set the other preferences as well

brianchandotcom committed:
LPD-28483 Wordsmith

brianchandotcom committed:
LPD-28483 Rename to match _notificationTemplateLocalService.addNotificationTemplate

ealonso committed:
LPD-X Moves Asset Publisher tests to content management team

ealonso committed:
LPD-X Removes tests

ealonso committed:
LPD-X Merged both methods

ealonso committed:
LPD-X Migrate to playwright

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
